src/mnode/src/mnodeTable.c

```diff
@@ -1225,7 +1225,7 @@ static int32_t mnodeProcessDropSuperTableMsg(SMnodeMsg *pMsg) {
 static int32_t mnodeFindSuperTableTagIndex(SSTableObj *pStable, const char *tagName) {
   SSchema *schema = (SSchema *) pStable->schema;
   for (int32_t tag = 0; tag < pStable->numOfTags; tag++) {
-    if (strcasecmp(schema[pStable->numOfColumns + tag].name, tagName) == 0) {
+    if (strcmp(schema[pStable->numOfColumns + tag].name, tagName) == 0) {
```

Comment:
Internally, the names of table names are required to be dealt with case-sensitively after parsing since now. For example, the tables of names DB and db should be recognized as different ones.